### ISSUE TITLE
Element type invalid when using Toaster

### ISSUE THREAD

**Issue by olup** [palantir/blueprint#2375]
### Bug report

- __Package version(s)__: 2.1.1
- __Browser and OS versions__: chrome, ubuntu 17

I get a 
```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `TransitionGroup`.
```
Every time I try to use the Toaster. As long as it has no children/toasts it's fine, but adding a child Toast or calling the show() method breaks the app.

Error comes from TransitionGroup built by Overlay when rendering in the Portal.

On create React App with a higly reconfigured admin-on-rest

**Comment by olup** [comment]
I am on react 16 indeed

**Comment by olup** [comment]
Actually I am confused and puzzled. The code had the same issue with
popover ; but then I can't say why it started working as I relaunched it
yesterday morning. No changes made. I had updated the day before from 2 to
2.1.1, and I manually npm installed react transition group. But it didn't
work, maybe something to do with the package lock ? Anyway, I started the
project again and it was gone. Pop over works fine. Toastser work fine with
.show() and the toaster and toast component way.

The code snippets were copies of the doc, that's what was so puzzling.

Right now and with not much more element around this we can close it
